### ISSUE TITLE
Nothing in modal after connexion

### ISSUE THREAD

**Issue by micker** [micz/ThunderAI#125]
**Thunderbird version and OS**
thunderbird 128.1 and windows 11

**ThunderAI version**
2.1.1

**AI Integration**
- [ x ] ChatGPT Web Interface
- [ ] OpenAI ChatGPT API
- [ ] Ollama API

**Describe the bug**
After connect to chatgpt nothing was display in modal
![image](https://github.com/user-attachments/assets/b29b926f-895e-43ac-96fa-93b730071ef4)
![image](https://github.com/user-attachments/assets/8cddaac8-38a1-46d7-a0b5-2b0ad450a1d2)
![image](https://github.com/user-attachments/assets/d0b017aa-7373-4f14-b245-adb0c414d2da)

thanks for any help

**Comment by micz** [comment]
Until you log in the ThunderAI does nothing.

Sometimes the page remains blank, but it's a ChatGPT problem.
Are you logging in with username and password or using a Google/Microsoft/Apple account?

**Comment by micker** [comment]
via google account (i didn't have problem in my browser

**Comment by micz** [comment]
The login workflow through an external account sometimes requires to open a popup, but that is not allowed by thunderbird.

I was not able to replicate that behaviour now.
Every time I try to login using a Google account, Thunderbird shows me the list of the authorized Google account and when I choose one, it logs in without problems.

Have you a Google account authorized as a Thunderbird Account to read the emails?

**Comment by micker** [comment]
yes i have a google account configured for reading email did you recommend to create a basic account ?

**Comment by micz** [comment]
Do you see any error in the Thunderbird console? 

You could try to login with email and password not using an external account.

**Comment by micker** [comment]
hello i try with direct email
how i can help you, where i can find console log to put here ?

**Comment by micz** [comment]
Press CTRL + SHIFT + J

**Comment by micker** [comment]
without any change i have this now
![image](https://github.com/user-attachments/assets/16ea696a-3e10-4c6d-ad86-51a769c21eca)
if i try to connect to my account
![image](https://github.com/user-attachments/assets/0cfa9d53-ea92-4f51-8192-f1536511cac6)

**Comment by micz** [comment]
This interface is the one used without login and it's completely different from the one for logged users.
So it will not finish loading automatically.
You need to click on "forcher l'achèvement".

The errors when trying to log in with your account are new to me, and they seem related to Thunderbird.
Are you trying to log in with username and password or clicking on Google Account?

**Comment by micker** [comment]
Yes i try with Google account with same result

**Comment by micz** [comment]
This is happening before ThunderAI can do anything, and I'm not able to reproduce the problem. I suggest trying to clear your cookies, but I can't be sure that this will resolve the issue.

I'm sorry, but I'm not sure how else to help you.

**Comment by micz** [comment]
Another user had the same problem and a workaround has been found.
See more details here: https://github.com/micz/ThunderAI/issues/155#issuecomment-2407926182
